examples/image_classifier/googlenet/README.md

```diff
@@ -0,0 +1,49 @@
+#### Sample commands to create a googlenet eager mode model archive, register it on TorchServe and run image prediction
+
+Run the commands given in following steps from the parent directory of the root of the repository. For example, if you cloned the repository into /home/my_path/serve, run the steps from /home/my_path/serve
+
+```bash
+wget https://download.pytorch.org/models/googlenet-1378be20.pth
+torch-model-archiver --model-name googlenet --version 1.0 --model-file ./serve/examples/image_classifier/googlenet/model.py --serialized-file googlenet-1378be20.pth --handler image_classifier --extra-files ./serve/examples/image_classifier/index_to_name.json
+mv googlenet.mar model_store/
+mkdir model_store
+mv googlenet.mar model_store/
+torchserve --start --model-store model_store --models googlenet
+curl http://127.0.0.1:8080/predictions/googlenet -T ./serve/examples/image_classifier/kitten.jpg
+```
+
+#### TorchScript example using googlenet image classifier:
+
+* Save the alexnet model in as an executable script module or a traced script:
+
+1. Save model using scripting
```

Comment:
@tigercosmos running the torchscript version, facing this [error](https://gist.github.com/HamidShojanazeri/f49b3f79a981d5b89bcf3cbac33c6f65) on post-processing. Wonder if you can repro, on your end.